person_capture/gui_app.py

```
@@ -5429,6 +5476,143 @@ def save_hit(
+                                                fallback_ratio,
+                                                (repair_bx1, repair_by1, repair_bx2, repair_by2),
+                                                anchor=((hfx1 + hfx2) * 0.5, (hfy1 + hfy2) * 0.5 + 0.18 * hfh),
+                                                min_size_xy=(max(cur_w, hfw * 1.18), max(cur_h, hfh * 1.22)),
+                                            )
+                                            guard_def = self._containment_deficit_xyxy(fixed, full_guard_box, margin_px=1.0)
+                                            if guard_def <= 0.01:
+                                                cx1, cy1, cx2, cy2 = fixed
+                                                ratio_str = fallback_ratio
+                                                c["ratio"] = fallback_ratio
+                                                fallback_done = True
+                                        except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)